fuzzers/071-ppips/generate.tcl

```diff
@@ -95,7 +95,13 @@ foreach tile_type {INT_L INT_R  BRAM_INT_INTERFACE_L BRAM_INT_INTERFACE_R \
             CLK_HROW_TOP_R CLK_HROW_BOT_R CLK_BUFG_TOP_R CLK_BUFG_BOT_R \
             IO_INT_INTERFACE_R IO_INT_INTERFACE_L RIOI3 LIOI3 LIOI3_TBYTETERM \
             RIOI3_TBYTETERM LIOI3_TBYTESRC RIOI3_TBYTESRC LIOI3_SING RIOI3_SING \
-            BRKH_INT HCLK_L HCLK_R} {
+            BRKH_INT HCLK_L HCLK_R HCLK_CMT \
+            CMT_TOP_L_UPPER_T CMT_TOP_L_UPPER_B \
+            CMT_TOP_L_LOWER_T CMT_TOP_L_LOWER_B \
+            CMT_TOP_R_UPPER_T CMT_TOP_R_UPPER_B \
+            CMT_TOP_R_LOWER_T CMT_TOP_R_LOWER_B \
+            INT_INTERFACE_L INT_INTERFACE_R \
+            HCLK_IOI3 CLK_BUFG_REBUF} {
```

Comment:
The `ppips` for `CLK_BUFG_REBUF` are wrong, remove it from this list.

Reply:
Ok, done. Why were they wrong?

Reply:
`071` isn't really designed for generic use.  The other ppips files look okay, but that tile is a little weird, and isn't really good to use with `071` tcl logic. I don't believe `CLK_BUFG_REBUF` has any ppips.